staging/src/k8s.io/legacy-cloud-providers/gce/gce.go

```diff
@@ -99,6 +99,12 @@ type Cloud struct {
 	// for the cloudprovider to start watching the configmap.
 	ClusterID ClusterID
 
+	// initializer is used for lazy initialization of some fields.
```

Comment:
shouldn't this be named subnetworkURLAndIsLegacyNetworkInitializer or similar?

Shouldn't the comment be more specific than "some fields"? :)